### ISSUE TITLE
Add "package" command to make something that can be shipped and run anywhere

### ISSUE THREAD

**Issue by maxandersen** [jbangdev/jbang#106]

**Comment by wkremser** [comment]
thank you for including my suggestion.

**Comment by quintesse** [comment]
Doesn't the "--native" option basically already give you that?

**Comment by maxandersen** [comment]
yes and no. `--native` attempts to create a native binary but that only works with dependencies that are "outofthebox" graalvm native-image friendly.

Idea from this one is to be able to have the `~/.jbang/cache` pre-created and ready to run...

probably need the `jbang build` command to just do the build and if you set the JBANG_CACHE_DIR you should be set to copy/ship it.

**Comment by quintesse** [comment]
Yeah, you're right of course that native isn't ready for that yet. So perhaps some kind of uber-jar solution? Or does that have drawbacks as well?

**Comment by vorburger** [comment]
#508 has something related to this specific to `--native` (I found this issue before creating that one by searching if this exists).

**Comment by maxandersen** [comment]
I Think we got uberjar, and ability to Build that caches so i think it is covered. 

Original tweet been deleted so cant check. Closing until convinced otherwise.
